stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/event/handler/internal/batch/BatchEvent.kt

```diff
@@ -17,11 +17,14 @@
 package io.getstream.chat.android.offline.event.handler.internal.batch
 
 import io.getstream.chat.android.client.events.ChatEvent
+import kotlin.math.absoluteValue
+import kotlin.random.Random
 
 /**
  * Events container to represent the source of the received events.
  */
 internal class BatchEvent(
+    val id: Int = Random.nextInt().absoluteValue,
```

Comment:
Do these strictly have to be unique or not?
I'm wondering if there's a chance of collision.